src/frequenz/sdk/actor/power_distributing/result.py

```diff
@@ -21,13 +22,13 @@ class _BaseResultMixin:
 
 @dataclasses.dataclass
 class _BaseSuccessMixin:
-    """Result returned when setting the power succeed for all batteries."""
+    """Result returned when setting the power succeed for all components."""
 
     succeeded_power: Power
     """The part of the requested power that was successfully set."""
 
-    succeeded_batteries: set[int]
-    """The subset of batteries for which power was set successfully."""
+    succeeded_components: abc.Set[int]
+    """The subset of components for which power was set successfully."""
```

Comment:
These changes should also go to the upgrading notes, as these types are now read-only.

Reply:
done here: https://github.com/frequenz-floss/frequenz-sdk-python/commit/b9c8573a